### ISSUE TITLE
how to install php-debugbar

### ISSUE THREAD

**Issue by trevorpan** [maximebf/php-debugbar#390]
Hi, I'm having a hard time installing the debugbar for SilverStripe 4.1.1.

Here is the terminal:

    composer require maximebf/debugbar

    Warning: Module 'intl' already loaded in Unknown on line 0
    Plugin command require-recipe (SilverStripe\RecipePlugin\RequireRecipeCommand) would   override a Composer command and has been skipped
    Plugin command update-recipe (SilverStripe\RecipePlugin\UpdateRecipeCommand) would override a Composer command and has been skipped
    Using version ^1.15 for maximebf/debugbar
    ./composer.json has been updated
    Loading composer repositories with package information
    Updating dependencies (including require-dev)
    Nothing to install or update
    Package tractorcow/recipe-plugin is abandoned, you should avoid using it. Use silverstripe/recipe-plugin instead.
    Generating autoload files

**Comment by barryvdh** [comment]
Perhaps check https://github.com/lekoala/silverstripe-debugbar for more information about how to integrate with SilverStripe.

**Comment by trevorpan** [comment]
Hi barryvdh, 

I have many times... it requires this debug bar to work. I've followed the instructions here of adding the requirements in the composer.json file and using the terminal command, but when I'm viewing the project as localhost:8888/tpan the debugbar does not appear. 

Using a mac mini, with Firefox Developer Edition.

**Comment by barryvdh** [comment]
Did you raise an issue there?

**Comment by trevorpan** [comment]
yes, https://github.com/lekoala/silverstripe-debugbar/issues/96 

debugbar is a requirement does not appear in the browser window. Do have any ideas why?
composer.json has it included, terminal command has been executed.

**Comment by trevorpan** [comment]
Hi barrybdh,

How do you typically use this file?

    <?php
    use DebugBar\StandardDebugBar;

    $debugbar = new StandardDebugBar();
    $debugbarRenderer = $debugbar->getJavascriptRenderer();

    $debugbar["messages"]->addMessage("hello world!");
    ?>
    <html>
        <head>
            <?php echo $debugbarRenderer->renderHead() ?>
        </head>
        <body>
            ...
            <?php echo $debugbarRenderer->render() ?>
       </body>
    </html>
